ui/components/component-library/avatar-network/avatar-network.js

```diff
@@ -14,70 +14,76 @@ import {
 } from '../../../helpers/constants/design-system';
 import { AVATAR_NETWORK_SIZES } from './avatar-network.constants';
 
-export const AvatarNetwork = ({
-  size = Size.MD,
-  name,
-  src,
-  showHalo,
-  color = TextColor.textDefault,
-  backgroundColor = BackgroundColor.backgroundAlternative,
-  borderColor = BorderColor.transparent,
-  className,
-  ...props
-}) => {
-  const [showFallback, setShowFallback] = useState(false);
+export const AvatarNetwork = React.forwardRef(
+  (
+    {
+      size = Size.MD,
+      name,
+      src,
+      showHalo,
+      color = TextColor.textDefault,
+      backgroundColor = BackgroundColor.backgroundAlternative,
+      borderColor = BorderColor.transparent,
+      className,
+      ...props
+    },
+    ref,
+  ) => {
+    const [showFallback, setShowFallback] = useState(false);
 
-  useEffect(() => {
-    setShowFallback(!src);
-  }, [src]);
+    useEffect(() => {
+      setShowFallback(!src);
+    }, [src]);
 
-  const fallbackString = name && name[0] ? name[0] : '?';
+    const fallbackString = name && name[0] ? name[0] : '?';
 
-  const handleOnError = () => {
-    setShowFallback(true);
-  };
+    const handleOnError = () => {
+      setShowFallback(true);
+    };
 
-  return (
-    <AvatarBase
-      size={size}
-      display={DISPLAY.FLEX}
-      alignItems={AlignItems.center}
-      justifyContent={JustifyContent.center}
-      className={classnames(
-        'mm-avatar-network',
-        showHalo && 'mm-avatar-network--with-halo',
-        className,
-      )}
-      {...{ backgroundColor, borderColor, color, ...props }}
-    >
-      {showFallback ? (
-        fallbackString
-      ) : (
-        <>
-          {showHalo && (
+    return (
+      <AvatarBase
+        ref={ref}
+        size={size}
+        display={DISPLAY.FLEX}
+        alignItems={AlignItems.center}
+        justifyContent={JustifyContent.center}
+        className={classnames(
+          'mm-avatar-network',
+          showHalo && 'mm-avatar-network--with-halo',
+          className,
+        )}
+        {...{ backgroundColor, borderColor, color, ...props }}
+      >
+        {showFallback ? (
+          fallbackString
+        ) : (
+          <>
+            {showHalo && (
+              <img
+                src={src}
+                className={
+                  showHalo ? 'mm-avatar-network__network-image--blurred' : ''
```

Comment:
This can be `className={classnames({ 'mm-avatar-network__network-image--blurred': showHalo })}`

Reply:
Hmm 🤔 . I can do it here but I believe it's better to update this in this [PR](https://github.com/MetaMask/metamask-extension/pull/18678) since we are updating avatar-components in [18678](https://github.com/MetaMask/metamask-extension/pull/18678). Wdyt?